### ISSUE TITLE
Failed downloaded

### ISSUE THREAD

**Issue by dafidxcode** [MichalDakowicz/instagram-post-downloader#1]
When i'm trying download with web app i've problem "Download failed: Video/file not found"

**Comment by MichalDakowicz** [comment]
are you trying to download a multi image/video post?

**Comment by dafidxcode** [comment]
yeah some videos, whereas when I run them from termux everything is fine
